src/Jobs/WorkflowTimeoutJob.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Symbiote\AdvancedWorkflow\Jobs;
+
+use Symbiote\AdvancedWorkflow\Actions\TimeoutTransitionInstance;
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+
+class WorkflowTimeoutJob extends AbstractQueuedJob
+{
+    public static $limit = 20;
+
+    public function getTitle()
+    {
+        return "Workflow Timeout Job";
+    }
+
+    public function getSignature()
+    {
+        return WorkflowTimeoutJob::class;
+    }
+
+    public function setup()
+    {
+        $ttis = TimeoutTransitionInstance::get();
```

Comment:
Will this also include the transition  instances for completed workflows?

Reply:
True, added obvious filter.